Merge Branch-UG2 Hong Liang with Master (#140)
AY2223S1-CS2103-F09-2/tp: docs/UserGuide.md

Format: `help`

### Adding a contact: `addPerson`

Adds a contact to the application.

Format: `addPerson n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS g/GENDER d/DOB`

- The compulsory parameters are: name (`n`), phone number (`p`), email (`e`), address (`a`), gender (`g`), date of birth(`d`).

> - The parameters in [ ] are optional parameters, including tag (`t`).

Reviewer: can remove this
```suggestion
- The parameters in [ ] are optional parameters, including tag (`t`).
```

> - A person in the contact list can have more than 1 tag.

Reviewer: can remove this line too
```suggestion
- A person in the contact list can have more than 1 tag.
```

Author: oh sorry i missed this!

- Edits the person at the specified `INDEX`. The index refers to the index number shown in the displayed person list.
  The index must be **a positive integer** 1, 2, 3, …​, and it must be within the range of the contact list index.

> - At least one of the optional fields must be provided.

Reviewer 2: Actually, this line should be kept since this is the edit command! All of the fields are optional when editing a person, just that at least one field must be present.

Author: ok thanks for catch have changed

  The index must be **a positive integer** 1, 2, 3, …​, and it must be within the range of the contact list index.

> - At least one of the optional fields must be provided.

Reviewer: can remove this line 142

Author: same as above

- Existing values will be updated to the input values.
- Date format accepted is: dd/mm/yyyy.
- When editing tags, the existing tags of the person will be removed (i.e. adding of tags is not cumulative).

> - You can remove all the person’s tags by typing t/ without specifying any tags after it.

Reviewer: can remove this line 145-146

Author: actually as qiaoran did the refactoring for edit command i left this here for her to remove, but ill remove it anyways

* `untagEvent 1 p/2` untags the 2nd person in the contact list from the 1st event in the event list
* `untagEvent 2 p/2 4 5` untags the 2nd, 4th, 5th person in the contact list from the 2nd event in the event list

> ### Generating piechart of statistics of the people tagged to an event in the event list : `makeStats`

Reviewer 2: ```suggestion
### Generating pie charts of statistics of the people tagged to an event: `makeStats`
```

Author: ah i see ok will change!
